pkg/hub/deployer/utils.go

```diff
@@ -0,0 +1,11 @@
+package deployer
+
+import (
+	"fmt"
+
+	appsapi "github.com/clusternet/clusternet/pkg/apis/apps/v1alpha1"
+)
+
+func FormatFeed(feed appsapi.Feed) string {
```

Comment:
No need for a new function

Reply:
utils to includes several function including FormatFeed, not only one function now